lambdas/backend-api/src/utils/schema-for.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod';
+
+export const schemaFor =
+  <Output, Input = Output>() =>
+  <S extends z.ZodType<Output, z.ZodTypeDef, Input>>(schema: S) =>
+    schema;
```

Comment:
isn't this just longhand for z.infer?

Reply:
I didn't want to use z.infer because the types are autogenerated. So I needed to way to enforce type correctness with z schemas. This is entirely ripped from comms-mgr. 

I think it's pretty neat TBH!